ios/Helpers/RCCTitleViewHelper.m

```diff
@@ -194,10 +194,17 @@ -(UILabel*)setupTitle:(NSDictionary*)style
     
     UIFont *titleFont = [UIFont boldSystemFontOfSize:17.f];
     
-    id fontSize = style[@"navBarTitleFontSize"];
```

Comment:
This change would be a breaking change for all API users. We should not break the API, especially for something like this, because it would break our users' apps.